website/source/docs/configuration/index.html.md

```diff
@@ -121,7 +121,7 @@ to specify where the configuration is.
   allows the decryption/encryption of raw data into and out of the security
   barrier. This is a highly privileged endpoint.
 
-- `ui` `(bool: false, Enterprise-only)` – Enables the built-in web UI, which is
```

Comment:
Do we want to highlight this as only avaliable in 0.10+?

Reply:
No -- our docs don't really indicate versions (except in a few places, which they aren't consistent about). A higher level versioned-docs project seems to be slowly gathering steam, which will address this.

Reply:
That said, if you want to note it, I'm fine with it.

Reply:
Cool, I think a brief note would be good, with the "real" fix coming with the versioned docs work in the future